lib/open_project/static/links.rb

```diff
@@ -189,6 +189,18 @@ def static_links
             },
             chargebee: {
               href: 'https://js.chargebee.com/v2/chargebee.js',
+            },
+            webinar_videos: {
```

Comment:
Thanks for adding new links to the static links. This will make it harder to forget about these links later and to avoid them becoming a dead link 👍